app/models/dashboard.rb

```diff
@@ -4,18 +4,34 @@ def initialize(user, defence_requests_scoped_by_policy)
     @defence_requests_scoped_by_policy = defence_requests_scoped_by_policy
   end
 
-  def defence_requests
-    ordered_defence_requests
-  end
-
   def user_role
     user.roles.uniq.first
   end
 
+  def defence_requests(type)
+    type == :active ? active_defence_requests : closed_defence_requests
+  end
```

Comment:
We shouldn't need to do this here - the controller should initialise the dashboard model with the correct collection of Defence Requests